packages/solutions/module-tools/src/utils/dts.ts

```diff
@@ -27,6 +27,18 @@ export const getProjectTsconfig = async (
   return json5.parse(fs.readFileSync(tsconfigPath, 'utf-8'));
 };
 
+export async function detectTSVersion(appDirectory?: string) {
+  // Detect typescript version from current cwd
+  // return the major version of react
+  const cwd = appDirectory ?? process.cwd();
+  const reactPath = join(cwd, 'node_modules', 'typescript');
```

Comment:
reactPath => tsPath?

Reply:
yes, you are right

Reply:
You can replace `reactPath` with the appropriate variable name.